keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -987,6 +985,12 @@ class SqlArtifactRepository(
       .where(ENVIRONMENT.NAME.eq(environment.name))
       .and(ENVIRONMENT.DELIVERY_CONFIG_UID.eq(uid))
 
+  private fun DeliveryConfig.getUidFor(environmentName: String): Select<Record1<String>> =
```

Comment:
Is there a specific reason to return this type here instead of just `String`? You can just add a `fetchOne(ENVIRONMENT.UID)` at the end of the query.

Reply:
Yes, this makes it so that you can use it to build another query, instead of making the call right then

Reply:
Ah, TIL! Thanks!